app/models/metric/purging.rb

```diff
@@ -35,6 +36,10 @@ def self.purge_realtime_timer(ts = nil)
     purge_timer(ts, "realtime")
   end
 
+  def self.purge_rollup_timer
```

Comment:
isn't this method overwriting the one on line 18?

Reply:
😱  - Nice find - I totally missed that.

Reply:
thanks so much. glad it was right there in this PR but I totally didn't see it